client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -37,6 +37,12 @@
 
 public class RssSparkConfig {
 
+  public static final ConfigOption<Boolean> RSS_CLIENT_BLOCK_ID_SELF_MANAGED_ENABLED =
+      ConfigOptions.key("rss.client.blockId.selfManagedEnabled")
```

Comment:
`selfManagementEnabled`?

Reply:
selfmanaged is adjective. Here the noun is more proper. 👍

Reply:
Maybe we should remove `client` word to align other config options.

Reply:
> Maybe we should remove client word to align other blockId config options.

Done

> Except, I'm a little worried about the config options about Uniffle. Because there are some less elegant issues. client seems extra. Don't use camel style. Proper segments.

This could be optimized because we have the discard mechanism in configOption, that need to be detailed in doc.